### ISSUE TITLE
glyphicons-halflings-regular.woff2 Failed to load resource: the server responded with a status of 404 (Not Found)

### ISSUE THREAD

**Issue by dhqcz** [Pingendo/pingendo-bootstrap#9]
Hello,

I use this CSS: http://pingendo.github.io/pingendo-bootstrap/themes/default/bootstrap.css

And there are not working fonts:

http://pingendo.github.io/pingendo-bootstrap/themes/fonts/glyphicons-halflings-regular.woff2 Failed to load resource: the server responded with a status of 404 (Not Found)
http://pingendo.github.io/pingendo-bootstrap/themes/fonts/glyphicons-halflings-regular.woff Failed to load resource: the server responded with a status of 404 (Not Found)
http://pingendo.github.io/pingendo-bootstrap/themes/fonts/glyphicons-halflings-regular.ttf Failed to load resource: the server responded with a status of 404 (Not Found)

Is it bug or may I upload CSS into my website and load fonts from my side too?

Thank you.

**Comment by FelixINX** [comment]
Same error. They aren't include in the theme. Must be fixed.

**Comment by Pingendo** [comment]
We should remove Glyphicons at all in favour of Font-awesome

**Comment by FelixINX** [comment]
I agree with this because Font Awesome is a bigger library that Glyphicons. But, there are some place that Font Awesome don't work with Bootstrap, for example, the controls of the Carousel.

**Comment by dandamudi** [comment]
i also got same issue, please help me if anybody have solution, i spent whole day on this issue

**Comment by Sparragus** [comment]
@Pingendo What's up with this?

**Comment by jayrocket** [comment]
@Pingendo Me too. It worked when using a FTP mut on my Mamp it doesn't.

**Comment by GongchuangSu** [comment]
@GeorgeMajor You can do this in HTML:
`<link href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap-glyphicons.css" rel="stylesheet">`

**Comment by codeCesar** [comment]
Had the same issue I was using local file. got rid of it and replaced it with cdn works fine now. Hope this helps someone, Happy coding
`<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">`

**Comment by sumanbadusu** [comment]
how to find that path to replace above given code to slove this error

**Comment by mounikakatt** [comment]
Create new fonts folder inside root project and then add fonts
it will be work

**Comment by Abduelhamit01** [comment]
> Create new fonts folder inside root project and then add fonts
> it will be work

Thank you! After that, I watched at my bootstrap.min.css file and saw "../fonts/" and knew I need to create the folder 1 folder back. Thank you!!
